tonic_textual/classes/record_api_request_options.py

```diff
@@ -0,0 +1,36 @@
+from typing import List, Optional
+
+
+class RecordApiRequestOptions(dict):
+    """
+    Class to denote whether to record an API request.
+
+    Parameters
+    ----------
+    record : bool
+        Whether to record the request.
+
+    retention_time_in_hours: int
+        The number of hours to store the request. The request is then purged automatically.
+
+    tags : Optional[List[str]]
+        An optional list of tags to assign to the request. Used to help search for the request on the API Explorer page.
+    """
+    def __init__(self, record: bool, retention_time_in_hours: int, tags: Optional[List[str]] = []):
```

Comment:
```suggestion
    def __init__(self, record: bool, retention_time_in_hours: int, tags: List[str] = []):
```

If `tags` is `None`, an `InvalidJsonForRedactionRequest` error is thrown.